src/features/lightspeed/utils/explorerView.ts

```diff
@@ -117,16 +124,45 @@ export function setWebviewMessageListener(
 ) {
   webview.onDidReceiveMessage(
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    (message: any) => {
+    async (message: any) => {
       console.log(
         `Lightspeed explorer - Message received: ${JSON.stringify(message)}}`,
       );
       const command = message.command;
 
       switch (command) {
-        case "connect":
+        case "connect": {
+          const ansibleSettings = workspace.getConfiguration("ansible");
+
+          const lightspeedEnabled =
+            await ansibleSettings.get("lightspeed.enabled");
+
+          const lightSpeedSuggestionsEnabled = await ansibleSettings.get(
+            "lightspeed.suggestions.enabled",
+          );
+
+          if (!lightspeedEnabled) {
+            await ansibleSettings.update(
+              "lightspeed.enabled",
+              true,
+              workspace.workspaceFolders
+                ? ConfigurationTarget.Workspace
+                : ConfigurationTarget.Global,
+            );
+          }
+
+          if (!lightSpeedSuggestionsEnabled) {
```

Comment:
IMO, if the value is already set to `true`, nothing bad will happen.

I don't think we need to read `lightSpeedSuggestionsEnabled` first. This increases the complexity of the code, and reduce the test coverage, without bringing any tangible value IMO.